site/config/custom_configuration.md

```diff
@@ -0,0 +1,34 @@
+## Custom Configuration
+
+WDT allows you to create or extend the pre-installed type definitions, model filters, variable injectors, and target environments. Starting with WDT 1.10.0, these additional configuration files can be stored outside the `$WLSDEPLOY_HOME/lib` directory. This allows the files to remain in place if the WDT installation is moved or upgraded to a new version.
+
+To use a separate configuration directory, set the `WDT_CUSTOM_CONFIG` environment variable to the directory to be used for configuration. For example:
+```
+export WDT_CUSTOM_CONFIG=/etc/wdtconfig
+```
+
+The customized configuration files should be named and organized the same way they would be under the `$WLSDEPLOY_HOME/lib` directory. For example:
+```
+/etc/wdtconfig
+    injectors
+        *.json  (injector files)
+    typedefs
+        *.json  (typedef files)
+    targets
+        my-target
+            target.json
+            *.py  (filter files)
+    model_filters.json
+    model_variable_injector.json
+    variable_keywords.json
+```
+This is a full set of files that can be configured, you will only need to add the files you have created or extended.
```

Comment:
configured, you -> configured; you (these are two sentences; if you want them joined, use a semi-colon.)
only need -> need only

Reply:
two sentences is better thx